src/model/neuroevolution/individual.py

```diff
@@ -228,9 +326,38 @@ def layer_swap_reproduction(
 
         return child_1, child_2
 
+    @staticmethod
+    def _get_random_layers_from_children(
+        *children: ChildIndividual,
+    ) -> Tuple[Layer, ...]:
+        """
+        Returns a randomly chosen layer for each provided child.
+
+        Choose a random layer index (for each provided child in `children`)
+        and returns the tuple containing provided children's layers at this index.
```

Comment:
The additional description is redundant, it also contains implementation details.

Reply:
Done